sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -42,6 +42,11 @@ export interface CreateKeyOptions extends coreHttp.OperationOptions {
     };
 }
 
+// @public
+export interface CreateOctKeyOptions extends CreateKeyOptions {
+    hsm?: boolean;
```

Comment:
1. This should've been `hardwareProtected`, but I see you shipped previously with `hsm` so that's fine. I guess this was missed in unification.
2. What about `keySize` here? That's valid as well. See https://github.com/Azure/azure-sdk-for-net/blob/f8aabfa3cc1b0a7ae2b6a1c111ea96416b3a4f4d/sdk/keyvault/Azure.Security.KeyVault.Keys/src/CreateOctKeyOptions.cs#L27. In .NET we have optional "parameters" as read-write properties instead of constructor params (reserved for required "parameters" and their properties are typically read-only).

Reply:
Opened https://github.com/Azure/azure-sdk-for-js/issues/13545 for the second issue.

Reply:
1. Agree on using `hardwareProtected` but yea I kept it `hsm` to be consistent across our APIs (see this comment: https://github.com/Azure/azure-sdk-for-js/pull/13522#discussion_r568756374)
2. I was wondering as well, but noticed we already offer `keySize` in `CreateKeyOptions`. See here: https://github.com/Azure/azure-sdk-for-js/blob/d476db9362d0ea071500467f4fc3680984a5c5e3/sdk/keyvault/keyvault-keys/src/keysModels.ts#L297

That change was before my time, but when I used this in sample code keySize was available to me. Do you think I should move it to a few specific Create<KeyType>Options classes instead?

Reply:
Interesting. In the PR, I noticed that `createRsaKeyOptions` also defines it at https://github.com/Azure/azure-sdk-for-js/blob/d476db9362d0ea071500467f4fc3680984a5c5e3/sdk/keyvault/keyvault-keys/src/keysModels.ts#L351, which is what threw me. Probably can't change it now, but `keySize` probably shouldn't have been defined on the base; practically, though, I doubt it would never *not* be a parameter.

Reply:
Yea, not sure why it's overwritten - might be a miss. But, since we can't remove `createKeyOptions.keySize` now maybe the thing to do is to remove the overrides since it's just debt at this point? I can reuse and update #13545 to reflect that.